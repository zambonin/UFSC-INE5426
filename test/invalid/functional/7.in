# wrong attribution between int array, 10 < 100
int t[100], output[10]
output = map(lambda int x -> x + 2, t)
