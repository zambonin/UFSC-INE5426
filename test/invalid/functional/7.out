[Line 9] semantic error: operation between mismatched array sizes
